.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+PR description here.
+
+### Todo
+
+- [ ] If you have more things to do
+- [ ] Add them here
+
+### Preview
+
+[https://deploy-preview-[PR number]--nubook.netlify.com/](https://deploy-preview-[PR number]--nubook.netlify.com/)
+
+Note: Update PR_number to the number on this PR and complete the link to the page you have updated/added. 
```

Comment:
So there's only one place to edit (the angle bracket syntax for links is useful when the link URL and text are the same):

```suggestion
<https://deploy-preview-[PR_NUMBER]--nubook.netlify.com/[PAGE_SLUG]>

Note: Update `[PR_NUMBER]` to the number on this PR and `[PAGE_SLUG]` with the "slug" of the page you have updated/added. You can find the slug at the top of the page in the markdown file.
```